### PR TITLE
Moved AppLifecycle samples out of Quarantine

### PR DESCRIPTION
**Pull request by andreww-msft**

### CODE REVIEW

File: Samples/AppLifecycle/Activation/cpp-console-unpackaged/CppWinRtConsoleActivation/main.cpp [hunk 1]
```diff
@@ -191,22 +191,23 @@ void UnregisterForActivation()
 
 void GetActivationInfo()
 {
-    AppActivationArguments args = AppInstance::GetCurrent().GetActivatedEventArgs();
+    AppActivationArguments args =
```
Comment (erik0686) [inline Samples/AppLifecycle/Activation/cpp-console-unpackaged/CppWinRtConsoleActivation/main.cpp:194]: Are these changes the same as the other PR?

**Comment by kythant** [comment]
Looks like there is a merge conflict. Think you'll have to pull from main first.

File: Samples/AppLifecycle/Activation/cpp-console-unpackaged/CppWinRtConsoleActivation/main.cpp (hunk 1 above)
Comment (andreww-msft) [reply Samples/AppLifecycle/Activation/cpp-console-unpackaged/CppWinRtConsoleActivation/main.cpp]: Yes.

**Review by kythant — APPROVED** [review]

**Pull request merged**